packages/circuit-ui/components/NotificationInline/NotificationInline.tsx

```diff
@@ -140,7 +140,7 @@ const actionButtonStyles = ({ theme }: StyleProps & ButtonProps) =>
     font-weight: bold;
     text-decoration-line: underline;
     color: ${theme.colors.black};
-    padding-bottom: calc(${theme.spacings.kilo} - ${theme.borderWidth.kilo});
+    padding-bottom: ${theme.borderWidth.kilo};
```

Comment:
Ah, I think I see. Maybe we can tweak the button's focus outline for this element instead?